### PR TITLE
Fix html errors on Home Page

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | I've removed HTML errors from homepage |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | Use the W3C validator on homepage. Before, there are errors, and after only few warnings remains |

> Note that https://github.com/PrestaShop/ps_linklist/pull/8 from `ps_linklist` module needs to be merged to remove 2 additional HTML errors.

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl [hunk 1]
```diff
@@ -5,7 +5,7 @@
       <ul>
         {foreach $linkBlock.links as $link}
           <li>
-            <a  id="{$link.id}"
+            <a id="{$link.id}"
```
Comment (maximebiloe) [inline themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl]: Can you make a line break before `id`, please?

File: themes/classic/modules/ps_mainmenu/ps_mainmenu.tpl [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
     {if $nodes|count}
-      <ul aria-labelledby="dLabel" data-depth="{$depth}" class="top-menu">
+      <ul data-depth="{$depth}" class="top-menu">
         {foreach from=$nodes item=node}
           {if $node.children|count}
             <li class="{$node.type}{if $node.current} current {/if}">
-              <a {if $depth === 0} data-toggle="dropdown" {/if}
-                 class = "{if $depth >= 0}dropdown-item{/if}{if $depth === 1} dropdown-submenu{/if}"
-                 href  = "{$node.url nofilter}"
+              <a class="{if $depth >= 0}dropdown-item{/if}{if $depth === 1} dropdown-submenu{/if}"
```
Comment (maximebiloe) [inline themes/classic/modules/ps_mainmenu/ps_mainmenu.tpl]: A line break is needed here too 😉

File: themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl (hunk 1 above)
Comment (Shudrum) [inline themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl]: Merged too fast…

Comment (mickaelandrieu) [inline themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl]: curious i did it, but in the related module :/

Comment (mickaelandrieu) [inline themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl]: btw, done https://github.com/PrestaShop/PrestaShop/commit/d022e1a2305abb2a055524bb37ab1718f0009674

**Pull request merged**